packages/devtools/src/index.js

```diff
@@ -41,6 +41,12 @@ export default class DevTools {
         const vendorCapPrefix = Object.keys(params.capabilities).find(
             (capKey) => availableVendorPrefixes.includes(capKey))
 
+        /**
+         * save original set of capabilities to allow to request the same session again
+         * (e.g. for reloadSession command in WebdriverIO)
+         */
+        params.requestedCapabilities = { ...params.capabilities }
```

Comment:
Do you think it's fine that nested objects are can be still changed? 
```
{
  browser: 'chrome',
  chromeOptions: { ... } // here
}
```

Reply:
I wouldn't see a reason why someone would still do that - I am afraid we face various of implications if someone just randomly re-assigns capability values. To fix this it would be out of scope of this PR though.